pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Tags'
+weight: 63
+---
+
+--- Unique Description about AsyncAPI document to be inserted here ---
+
+A `tag` is a label or category that helps you logically group related objects such as channels or servers within an event-driven system. The `tags` object provide a structured way to group channels, operations or other components based on their functionality, purpose and/or any other relevant criteria.
+
+The `tags` object serves as a container for defining and managing tags in a systematic manner. Within the `tags` object, you can define individual tags, specifying a name and providing an optional description that adds further context to the tag's purpose or usage. 
```

Comment:
> The `tags` object serves as a container for defining and managing tags in a systematic manner

what do you mean by that? in the end it is just and only about grouping common parts of your system